### DISCUSSION TITLE
How to fix the [MISSING TRANSLATION] error message?

### DISCUSSION THREAD

**Discussion by ianpiper** [GetPublii/Publii#1856] [Help]
I have just upgraded my version of the Documentation theme from 1.4 to 3.0. I haven't yet uploaded the site to my server because I'm seeing an issue when previewing. On every page, when I begin to scroll, I see a popup with the message "[MISSING TRANSLATION]".  I also see this on the tags list page at the top of the page.
I saw an earlier discussion where this problem seemed to be fixed by checking the [theme].lang.json file. I've checked this and it seems OK (documentation.lang.json)
My version of Publii is 0.46.
Can anyone suggest what may be wrong?

**Comment by michaelfluegge** [comment]
Documentation is a paid theme, isn't it? I think it is better to ask your question at the "Premium Support" site

**Comment by gpsblues** [comment]
Hi,
you should check the strings in the themeName.lang.json file, probably the updated version of the theme uses different entries.
https://getpublii.com/docs/translate-publii-theme-to-another-language.html#thepubliithemelanguagefile
